### DISCUSSION TITLE
Cookies settings

### DISCUSSION THREAD

**Discussion by rtesse** [squidfunk/mkdocs-material#3817] [Q&A] [answered]
Hello, 

I have a question regarding the cookies with mkdocs-material. I defined an extra key with all the information as described in the documentation, but I don’t have the cookie consent form rendered. Here is the snippet of the code:

![image](https://user-images.githubusercontent.com/26649180/163363331-af381208-73d7-4fbd-88ac-f1ac102687ac.png)

I also add a link to change the cookies setting but it is not working. 

<img width="175" alt="image" src="https://user-images.githubusercontent.com/26649180/163363539-6ba9795e-2fdc-49a1-a581-a0a2f07a7589.png"> --> Nothing happens

Could you please tell me what I am doing wrong?

**Comment by squidfunk** [comment, marked as answer]
Are you using [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/)? The cookie consent is currently only available to [sponsors](https://squidfunk.github.io/mkdocs-material/insiders/#how-to-become-a-sponsor).

**Reply by rtesse** [reply]
Arg, sorry, I didn’t see that. I dont’t think our departement is an insider. I will check. Thanks for the answer.
